app/uk/gov/nationalarchives/omega/editorial/models/package.scala

```diff
@@ -123,6 +303,60 @@ package object editSets {
         |      "oci" : "COAL.2022.V3RJW.P",
         |      "scopeAndContent" : "Bedlington Colliery, Newcastle Upon Tyne. Photograph depicting: view of pithead baths.",
         |      "coveringDates" : "1960"
+        |    },
+        |    {
+        |      "ccr" : "COAL 80/80/3",
+        |      "oci" : "COAL.2022.V4RJW.P",
+        |      "scopeAndContent" : "Bedlington Colliery, Newcastle Upon Tyne. Photograph depicting: view of pithead baths.",
+        |      "coveringDates" : "1960"
+        |    },
+        |    {
+        |      "ccr" : "COAL 80/80/3",
```

Comment:
Can we please increment the CCRs as well as the OCIs so that they appear as distinct records. The CCRs and OCIs for both the records and the edit set should tie up so that we can see clearly they are separate records when testing and demo-ing.